tests/subsys/bootloader/bl_validation_ff_key/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
2020

everywhere (haven't checked)

Reply:
2018 on most of the .c/.h files so I guess could update those :) would put this under nit:

Reply:
Done